pkgs/development/compilers/jetbrains-jdk/jcef.nix

```diff
@@ -73,6 +73,25 @@ let rpath = lib.makeLibraryPath [
 ];
 
 buildType = if debugBuild then "Debug" else "Release";
+lookupTable = {
+  "x86_64-linux" = {
+    clang_format_version = "dd736afb28430c9782750fc0fd5f0ed497399263";
+    clang_format_sha256 = "sha256-4H6FVO9jdZtxH40CSfS+4VESAHgYgYxfCBFSMHdT0hE=";
+    cef_platform = "linux64";
+    cef_sha256 = "sha256-r+zXTmDN5s/bYLvbCnHufYdXIqQmCDlbWgs5pdOpLTw=";
+  };
+  "aarch64-linux" = {
+    clang_format_version = "dd736afb28430c9782750fc0fd5f0ed497399263";
+    clang_format_sha256 = "sha256-4H6FVO9jdZtxH40CSfS+4VESAHgYgYxfCBFSMHdT0hE=";
+    cef_platform = "linuxarm64";
+    cef_sha256 = "sha256-gCDIfWsysXE8lHn7H+YM3Jag+mdbWwTQpJf0GKdXEVs=";
+  };
+};
+
+cef_platform = lookupTable.${builtins.currentSystem}.cef_platform;
```

Comment:
Can't use `builtins.currentSystem` in pure contexts like flakes.

`stdenv.system` instead?